menu.css

```diff
@@ -0,0 +1,35 @@
+.menu-wrap {
+	background-color: rgba(0, 214, 170, .75);
+	position: fixed;
+	text-align: center;
+	z-index: 1;
+	width: 100%;
+}
+
+nav {
+	position: absolute;
+	text-align: left;
+	top: 100%;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/burger-one/pullRequest?prid=6050239)